### ISSUE TITLE
Faucet can't be installed with pip3

### ISSUE THREAD

**Issue by gizmoguy** [REANNZ/faucet#497]
Due to how we parse our version number in setup.py we break python3 compat:

```
root@6b45c889c8f6:/tmp# pip3 install ~/faucet/
Processing /root/faucet
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-5apv5jb1-build/setup.py", line 9, in <module>
        readme_contents = open(os.path.join(setup_dir, 'README.rst')).read()
      File "/usr/lib/python3.5/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 15736: ordinal not in range(128)
```

**Comment by gizmoguy** [comment]
So I've fixed the issue in faucet's setup.py that prevents installation with pip3 in #535.

Unfortunately it looks like there is the same bug present in PBR:

https://github.com/openstack-dev/pbr/blob/4c8bd90846bf4864f98c526659de5a7311ae192a/pbr/util.py#L315

We can work-around by not setting "description-file" in setup.cfg... or someone should work out how to lodge a bug against PBR, I see they don't take github issues.

**Comment by anarkiwi** [comment]
I vote for the workaround - as a TODO, since looks like there's no
deterministic process for a fix and it doesn't impact our functionality
(other than to install nicely!).

**Comment by gizmoguy** [comment]
Agree.

Fixed in #539
